rmf_traffic_ros2/test/unit/test_ParticipantRegistry.cpp

```diff
@@ -49,29 +49,35 @@ SCENARIO("Test idempotency of shape type")
   REQUIRE_THROWS(shape_type(node["type"]));
 }
 
-
-bool operator==(const rmf_traffic::Profile p1, const rmf_traffic::Profile p2)
+namespace rmf_traffic {
+bool operator==(const Profile p1, const Profile p2)
 {
   return rmf_traffic_ros2::convert(p1) == rmf_traffic_ros2::convert(p2);
 }
-
+namespace schedule {
```

Comment:
Doesn't the linter want a new line after the namespace declaration and before its closing bracket?

Reply:
Seems like the linter skips over namespaces. I'll go ahead and do that.